src/main/kotlin/blackjack/Fetcher.kt

```diff
@@ -0,0 +1,5 @@
+package blackjack
+
+fun interface Fetcher<A, B> {
+    fun fetch(args: A): B
```

Comment:
```suggestion
    operator fun invoke(args: A): B
```
이렇게 사용해 보는건 어떨까요? 사용하는 쪽에서 좀 더 간결하게 표현 가능할것 같아요.